vcs.go

```diff
@@ -822,6 +822,15 @@ var vcsPaths = []*vcsPath{
 		check:  noVCSSuffix,
 	},
 
+	// Gitlab
```

Comment:
hmmm...have you tested this?

looking at the code upstream, it looks like gitlab falls down the generic codepath:
https://github.com/golang/go/blob/master/src/cmd/go/internal/vcs/vcs.go#L1514

Reply:
it would be good to have a test that uses https://gitlab.com/nicks6/tilt-extension-experiment (or something similar) in get_test.go (which also has test for tilt-dev/tilt-extensions

Reply:
The general regex will work for gitlab only if the VCS path ends with `.git`

- `https://gitlab.com/nicks6/tilt-extension-experiment.git` will work but isn't clear that this is required
- with `https://gitlab.com/nicks6/tilt-extension-experiment` you will see
```
Error: cannot load ext://tilt_extension: loading extension repo tilt-extension-experiment: download error:
waiting 5s before retrying. Original error: dynamic lookup required to find mapping
```

I'll push up some tests in `get_test.go` for this too 👍